cognite_toolkit/cognite_modules/infield/cdf_infield_location/data_models/infield_apm_app_config.node.yaml

```
@@ -0,0 +1,22 @@
+autoCreateDirectRelations: true
+skipOnVersionConflict: false
+replace: true
+nodes:
+- space: cognite_app_data # should we have a separate space to avoid users overwriting config?

```

@danlevings I believe @erikangelsen-cognite had a concern regarding allowing users access to this space and the risk of corrupting the config

---

```suggestion
- space: cognite_app_config_data

```

Talked to Dan and we want to add a new space for holding config data